packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/Filtering/FilterFlyout.js

```diff
@@ -305,7 +323,12 @@ const FilterFlyout = ({
       return (
         <Dropdown
           {...components.Dropdown}
-          selectedItem={filtersState[column].value}
+          selectedItem={
+            filtersState[column].value === ''
+              ? 'Any'
+              : filtersState[column].value === ''
```

Comment:
Sorry this should be like this
```suggestion
              : filtersState[column].value
```